src/express/commands/chaos.js

```diff
@@ -19,10 +19,10 @@ async function removeAllPeers(ip, staticNodes) {
 
   const tasks = []
   for (let i = 0; i < staticNodes.length; i++) {
-    command = `/home/ubuntu/go/bin/bor attach ~/.bor/data/bor.ipc --exec "admin.removeTrustedPeer('${staticNodes[i]}')"`
+    command = `/home/ubuntu/go/bin/bor attach /var/lib/bor/data/bor.ipc --exec "admin.removeTrustedPeer('${staticNodes[i]}')"`
```

Comment:
Let's change `/home/ubuntu/go/bin` to `~/go/bin`

Reply:
Done, [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/80a4181e29019ffe3b0d5454b56f13689cff4f33).